01-fundamentals/first_letter.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2016 Ami . All rights reserved.
+
+def get_user_password():
+    password = input("Tell me your password:")
+    return password
+
+
+def show_first_letter_of(password):
```

Comment:
I meant the name of the function is more suitable as `print_...` instead of `show_...`.